ios/brave-ios/Sources/Brave/Frontend/Browser/HomePanel/NTPDataSource.swift

```diff
@@ -230,6 +234,16 @@ public class NTPDataSource {
       replaceFavoritesIfNeeded?(superReferralImageData.topSites)
     }
   }
+
+  private func hasGracePeriodEnded(_ sponsoredImageData: NTPSponsoredImageData) -> Bool {
+    guard let gracePeriod = sponsoredImageData.gracePeriod?.doubleValue,
+      let installDate = Preferences.DAU.installationDate.value
```

Comment:
`Preferences.DAU.installationDate` will reset after 14 days, if you need to ensure this is actually the install date, please use `Preferences.P3A.installationDate`

Reply:
Fixed

Reply:
@tmancey No

Reply:
:-( P3A it is then :-)